src/support/hazard.c

```diff
@@ -295,7 +295,7 @@ __wt_hazard_check(WT_SESSION_IMPL *session, WT_REF *ref, WT_SESSION_IMPL **sessi
     WT_CONNECTION_IMPL *conn;
     WT_HAZARD *hp;
     WT_SESSION_IMPL *s;
-    uint32_t i, j, hazard_inuse, max, session_cnt, walk_cnt;
+    uint32_t j, hazard_inuse, max, walk_cnt;
```

Comment:
Now that we've removed `i` I'm considering updating the name of variable `j` to `i`. I'll do it after the first review pass if you agree

Reply:
Yeah I agree.

Reply:
Done in [9bf5269](https://github.com/wiredtiger/wiredtiger/pull/9295/commits/9bf5269835a30930312cf1a44a9dc084176b3792)